plugins/inspect/os_inspector.rb

```diff
@@ -92,6 +92,11 @@ def get_os_from_os_release
     end
     if result["pretty_name"]
       result["pretty_name"] = strip_arch_from_name(result["pretty_name"])
+
+      # remove always changing Tumbleweed version from pretty name
+      if result["pretty_name"].include?("Tumbleweed")
```

Comment:
I'd prefer to have make the ckeck a bit more specific, e.g. 

```
if result["pretty_name"] =~ /^openSUSE.*Tumbleweed$/
```

or something like that.